explorer/client/package.json

```diff
@@ -2,9 +2,6 @@
   "name": "@chainlink/explorer-client",
   "private": true,
   "version": "0.0.1",
-  "description": "LINK Explorer Client",
```

Comment:
Why did we remove this + the other fields?

Reply:
Bringing it inline with the `feeds_ui/package.json`. The license is in the root `package.json` & the other fields don't really feel like they add any value.

Reply:
Fair enough, I guess we only need these fields if they're public packages